src/modules/services-systemd/services-systemd.conf

```diff
@@ -3,20 +3,20 @@
 #
 # Systemd services manipulation.
 #
-# This module can enable services, timers and targets for systemd
-# (if packaging doesn't already do that). It can also
-# disable services and targets as well as mask units.
+# This module can enable units (like services, sockets, paths, etc.) for
+# systemd (if packaging doesn't already do that). It can also disable or mask
+# units.
 #
-# The order of operations is fixed.  Enable services, enable targets,
-# enable timers, disable services, disable targets and finally apply masks.
+# The order of operations is fixed. Enable units, disable units, and finally
+# apply masks.
 ---
 
-# There are several configuration keys for this module:
-# *services*, *targets*, *timers*, *disable*, *disable-targets* and *mask*.
+# There are three configuration keys for this module:
+# *enable*, *disable*, and *mask*.
 # The value of each key is a list of entries. Each entry has two keys:
-#   - *name* is the (string) name of the service or target that is being
-#     changed. Use quotes. Don't include unit suffix in the name. For
-#     example, it should be "NetworkManager", not "NetworkManager.service"
+#   - *name* is the (string) name of the unit that is being changed.
+#     Use quotes. Please include unit suffix in the name. For example,
```

Comment:
I don't think we need to require the unit suffix in the name.  Why not leave it up to the distro if they want to include the suffix or not?

Reply:
> I don't think we need to require the unit suffix in the name. Why not leave it up to the distro if they want to include the suffix or not?

How do we resolve `cups.service` from `cups.socket` if the configuration only specifies `cups`? Send it directly to `systemctl`?